tools/run_tests/dockerize/build_interop_image.sh

```diff
@@ -67,8 +67,14 @@ else
   BASE_IMAGE=${BASE_NAME}_base:`md5 -r tools/dockerfile/interoptest/$BASE_NAME/Dockerfile | cut -f1 -d\ `
 fi
 
-# Make sure base docker image has been built. Should be instantaneous if so.
-docker build -t $BASE_IMAGE --force-rm=true tools/dockerfile/interoptest/$BASE_NAME || exit $?
+if [ "$DOCKERHUB_ORGANIZATION" != "" ]
+then
+  DOCKER_IMAGE_NAME=$DOCKERHUB_ORGANIZATION/$BASE_IMAGE
```

Comment:
FTR, I think this pulls `grpctesting/grpc_interop_python_base:8ec9e1c14119ef2d698394af491ff9983931a3f3`

It should try to pull `grpctesting/grpc_interop_python_8ec9e1c14119ef2d698394af491ff9983931a3f3`

besides this, it seems correct.